src/assets/icons/CloseIcon.tsx

```diff
@@ -4,16 +4,10 @@ import { createIcon } from '@chakra-ui/react';
 export const CloseIcon = createIcon({
```

Comment:
I didn't notice that earlier - we have the icons under `/components/icons`, can you move it there?

Reply:
ohhh la la. True! I will do. Thanks for noticing it.